staff/dbenavente/06-extra-exercises/7.js

```diff
@@ -0,0 +1,15 @@
+/*
+encodeWord
+Define a function called ​encodeWord​ that receives a string and return 
+the codified version of that string replacing the following characters:
+*/
+
+function encodeWordPlus(string){
+    string= string.replace('T',7)
```

Comment:
Try to NEVER modify directly the argument received. Create another clean one to work in the body of your function